packages/fxa-content-server/app/scripts/templates/test_template.mustache

```diff
@@ -37,20 +37,21 @@
     <div class="input-row">
       <input type="tel" id="phone" />
     </div>
-    <div class="input-row password-row">
-      <input type="password" class="password" id="password" pattern=".{8,}" data-synchronize-show="true" required />
+    <div class="tooltip-container">
+      <input type="password" id="password" pattern=".{8,}" data-synchronize-show="true" required />
+      <div id="password-strength-balloon-container"></div>
     </div>
-    <div class="input-row password-row">
-      <input type="password" class="password check-password tooltip-below" id="vpassword" pattern=".{8,}" required />
-      <div class="input-help input-help-focused input-help-complete-password">Must be at least 8 characters</div>
+    <div class="tooltip-container">
+      <input type="password" class="tooltip-below" id="vpassword" pattern=".{8,}" required />
+      <div>Must be at least 8 characters</div>
     </div>
-    <div class="input-row password-row">
+    <div class="tooltip-container">
       <input type="password" id="old_password" placeholder="Old password" required pattern=".{8,}" />
       <input type="password" id="new_password" placeholder="New password" required pattern=".{8,}" />
-      <div class="input-help input-help-focused input-help-signup">Must be at least 8 characters</div>
-      <div class="input-help input-help-forgot-pw links"><a href="/" class="reset-password">Forgot password?</a></div>
+      <div>Must be at least 8 characters</div>
+      <div class="links"><a href="/" class="reset-password">Forgot password?</a></div>
```

Comment:
I feel like this test file could be cleaned up further but it's a real PITA working with it and I did what was obvious. 🤷‍♀️